lib/utils/testLauncherHelper.js

```diff
@@ -166,15 +149,15 @@ function getDeviceLogStream(device, userArgs, startTime, logDir) {
  * Verify the debug port parameter
  *
  * @param {Object} ownArgw - implicitly derived launched parameters
- * @param {String} runMode - Suitest test execution mode (interactive|automated)
- * @returns {String}
+ * @param {string} sessionType - Suitest test execution mode (interactive|automated)
+ * @returns {number|undefined}
  */
-function getDebugPort(ownArgw, runMode) {
+function getDebugPort(ownArgw, sessionType) {
 	const {inspect, inspectBrk} = ownArgw;
 	const debugPort = inspectBrk || inspect;
 
 	/* istanbul ignore if  */
-	if (debugPort && runMode === AUTOMATED) {
+	if (debugPort && sessionType === AUTOMATED) {
```

Comment:
I think we need to have it well tested, Is it work with webstorm debugger? If i'm remember correctly it launches process with options.execArgv not args

Reply:
well, this particular change should not break how it works now, I did not test debugging in webStorm, I thought you did recently, or @JeKachan ?

Reply:
I think debugging will be working as it was but will it handle case when mode is AUTOMATED and inspect option is set not as launcher arg

Reply:
How else can it be set? ownArgw in this function argument will also receive all config options passed from rc file , user config file, cli args and default config. Maybe we should rename it to be more clear.

Reply:
I'm not 100% sure but webstorm and user may set it as execArgv https://nodejs.org/docs/latest/api/process.html#process_process_execargv

Reply:
Yeah looks like we don't handle this case